jetbrains-core/src/software/aws/toolkits/jetbrains/ToolkitPlaces.kt

```diff
@@ -0,0 +1,10 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains
+
+object ToolkitPlaces {
+    const val EXPLORER_TOOL_WINDOW = "ExplorerToolWindow"
+    const val EDITOR_PSI_REFERENCE = "Editor"
+    const val INTENTION_ACTION = "IntentionAction"
```

Comment:
Is this variable used somewhere?

Reply:
it was, then i completely deleted all of that since it didn't really make sense